src/Tools/dotnet-monitor/CollectionRules/Configuration/CustomShortcutsConfigureNamedOptions.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Actions;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Triggers;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Configuration
+{
+    internal sealed class CustomShortcutsConfigureOptions :
+        IConfigureOptions<CustomShortcutOptions>
```

Comment:
This should be an IPostConfigureOptions<T>. You cannot assume in an IConfigureOptions<T> implementation that your instance happens to run after the first instance that populates the options from binding; that is the purpose of IPostConfigureOptions, to change the options after they've been populated from their initial source(s).

Reply:
Related question (which was my original logic behind doing it this way) - can we assume that `CustomShortcutsPostConfigureNamedOptions` would run before `CollectionRulePostConfigureNamedOptions`?